### PR TITLE
Add airflow to helm tools

### PR DESCRIPTION
**Pull request by s-block**
## What

This PR adds the functionality to deploy a loc al version of the helm chart `airflow-sqlite`.

It is deployed in the users namespace and uses the `auth-proxy` for authentication.

## How to review

1. Go to tools
2. Click to deploy airflow
3. Click the link to see if airflow is accessible

https://trello.com/c/utqhVsoG

### CODE REVIEW

File: control_panel_api/tools.py [hunk 1]
```diff
@@ -119,6 +120,32 @@ def deploy_params(self, user):
         ]
 
 
+class Airflow(BaseTool):
+    name = 'airflow-sqlite'
+
+    def release_name(self, username):
+        return f'{username}-{self.name}'
```
Comment (andyhd) [inline control_panel_api/tools.py]: This override does nothing - you can remove it

File: control_panel_api/tools.py [hunk 2]
```diff
@@ -119,6 +120,32 @@ def deploy_params(self, user):
...
+
+    @property
+    def chart_name(self):
+        return f'mojanalytics/{self.name}'
```
Comment (andyhd) [inline control_panel_api/tools.py]: This is also overriding the base class unnecessarily

File: control_panel_api/tools.py (hunk 1 above)
Comment (s-block) [reply control_panel_api/tools.py]: removed

File: control_panel_api/tools.py (hunk 2 above)
Comment (s-block) [reply control_panel_api/tools.py]: removed

File: control_panel_api/settings/base.py [hunk 3]
```diff
@@ -177,6 +177,12 @@ def is_enabled(value):
 JUPYTER_LAB_AUTH_CLIENT_ID = os.environ.get('JUPYTER_LAB_AUTH_CLIENT_ID')
 JUPYTER_LAB_AUTH_CLIENT_SECRET = os.environ.get('JUPYTER_LAB_AUTH_CLIENT_SECRET')
 
+AIRFLOW_AUTH_CLIENT_DOMAIN = os.environ.get('AIRFLOW_AUTH_CLIENT_DOMAIN', OIDC_DOMAIN)
```
Comment (xoen) [inline control_panel_api/settings/base.py]: nice 👍

**Pull request merged**